### PR TITLE
CESYS-10 feat: patching PP to work with CE loop tests

### PR DESCRIPTION
**Pull request by makokal**

### CODE REVIEW

File: launch/figure_eight.launch [hunk 1]
```diff
@@ -1,11 +1,31 @@
 <launch>
  
+  <!-- Chuck Stack -->
```
Comment (dnikitop) [inline launch/figure_eight.launch]: probably don;'t need these nodes

Comment (makokal) [reply launch/figure_eight.launch]: Yes, this was for local testing only

Comment (makokal) [reply launch/figure_eight.launch]: Removed them

**Review by makokal — COMMENTED** [review]
LGTM

**Pull request merged**